docusaurus/docs/how-to-guides/data-source-plugins/profile-backend-plugin.md

```diff
@@ -0,0 +1,145 @@
+---
+id: profile-backend-plugin
+title: Profile a backend plugin
+description: How to profile a backend plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - profile
+  - profiling
+  - continuos profiling
+  - backend
+  - back-end
+---
+
+This guide provides instructions for configuring a backend plugin to enable certain diagnostics when it starts, generating _profiling data_. Profiling data provides potentially useful information
+for investigating certain performance problems, such as high CPU or memory usage, or when you want to use [continuous profiling](https://grafana.com/oss/pyroscope/).
+
+## Configure profiling data
+
+The [Grafana configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/) allows you to configure profiling under the `[plugin.<plugin ID>]`.
+
+### Specify the plugin ID
+
+In this section of the file, specify the `<plugin ID>`, a unique identifier, for the backend plugin you want to profile, for example, [grafana-github-datasource](https://grafana.com/grafana/plugins/grafana-github-datasource/):
+
+```ini title="custom.ini"
+[plugin.<plugin ID>]
+profiling_enabled = true
+profiling_port = 6060
+profiling_block_rate = 5
+profiling_mutex_rate = 5
```

Comment:
Maybe also add these in the "Configure Grafana" page ?

https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/

Reply:
Good idea, but I rather link to here from there

Reply:
Or the other way around :)

Reply:
Yes that makes sense, I think linking to this page from the docs is a good idea!